src/styles/app.scss

```diff
@@ -1,2 +1,7 @@
 @import './nhsuk-8.1.1.min.css';
```

Comment:
We could import the nhsuk scss instead of the .css file by doing '../../node_modules/nhsuk-react-frontend/packages/nhsuk.scss' (this scss gets bundled up when the site is built)

You should then have access to the variables in `_colours.scss`. Background colour could be set like so; `background-color: $nhsuk-border-color`. 

Alternatively I've been following a pattern of just naming the colours I need directly in the file instead. So for the background colour I do;

```
$nhsuk-border-color: #d8dde0
```

Just so we know what the colour relates to.

Reply:
Done